app/models/template.rb

```diff
@@ -31,10 +31,6 @@ def self.template_includes
     []
   end
 
-  def preview_host_collection
```

Comment:
@timogoebel I don't think anyone wants to mess out with Remote Execution... you could never know when your schedule is next....